people/Hangmster/page.en.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Hangmster
+info:
+    alias: 仓鼠
+    location: Shijiazhuang, Hebei
+---
+
+Hangmster is an ally of us.
+
+They often interacted with friends at Twitter/X, and they were a barrel of laughs. They would also give warm hugs when we needed, console us when we were in anxiety, and mourn for our departed friends.
```

Comment:
They often interacted with friends on Twitter/X, bringing us laughter.
They gave warm hugs when we needed, consoled us when we were anxious, and mourned for our departed friends.